get_fb_comments_from_fb.py

```diff
@@ -143,7 +143,8 @@ def scrapeFacebookPageFeedComments(page_id, access_token):
 
             for status in reader:
                 has_next_page = True
-
+                after = ''
```

Comment:
@bakakaldsas, I'm not convinced that this solves the issue. Unless I'm mistaken, this will duplicate high-level comments without reaching the lower-level comments. If the lower-level comments cannot be accessed, that's one thing, but it would be better to simply skip the lower level comments (possibly writing a log of which comments were skipped) than to add the additional problem of duplication.